vagrant/k8s/service-monitors.yaml

```diff
@@ -0,0 +1,95 @@
+---
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: receiver-mock
+  namespace: sumologic
+  labels:
+    release: collection  # ensure this matches the `release` label on your Prometheus pod
```

Comment:
Nitpick: here we're using 2 spaces before the `#` starting the comment, elsewhere it's just 1. Maybe we establish one way or another and stick with it?

Reply:
Good point, I'm fan of python comments style, but we should stick to one common
@perk-sumo  any thoughts?

Reply:
I'd go with one space after.